pilot/pkg/security/authz/builder/testdata/v1beta1-simple-policy-td-aliases-in.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: httpbin
+  namespace: foo
+spec:
+  selector:
+    matchLabels:
+      app: httpbin
+      version: v1
+  rules:
+    - from:
+        - source:
+            principals: ["cluster.local/ns/rule[0]/sa/from[0]-principal[0]"]
```

Comment:
The namespace doesn't matter here and It's more cleaner to put the rule/from/principal together, i,e, cluster.local/ns/foo/sa/rule[0]-from[0]-principal[0], same for other fields.